samples/xlsx-import+svelte/README.md

```diff
@@ -0,0 +1,47 @@
+# Svelte
+
+This is an example how to use `xlsx-import` in Svelte.
+
+**LIB Version:** `2.3.4`
+
+## Scripts
+
+`npm run dev` - compiles and hot-reloads for development
+
+`npm run build` - compiles and minifies for production
+
```

Comment:
Could I ask you to add following scripts here?

* `npm run validate`
* `npm run cy:open`

After all, the list should contains: `start`, `build`, `validate`, `test`, and `cy:open`.